packages/cypress-commands/src/setups/enableNetworkShim/stubRequests.js

```diff
@@ -75,3 +84,28 @@ function getRequesStubResponseBody({
 
     return JSON.parse(responseBody)
 }
+
+function registerMissingRequestStub(state, { method, path, testName }) {
+    /*
+     * A missing requestStub could indicate there is a problem, but it
+     * could also be that an in-test cy.intercept is providing a static
+     * response object or fixture. Throwing an error here is problematic
+     * for the latter case, because the test would fail due to the error.
+     * So instead we collect missingRequestStubs on the state and let
+     * the plugin warn the user about this once the full run is completed.
+     */
+    if (!state.missingRequestStubs) {
+        state.missingRequestStubs = []
+    }
+
+    const isDuplicate = state.missingRequestStubs.some(
+        stub =>
+            stub.method === method &&
+            stub.path === path &&
+            stub.testName === testName
+    )
+
+    if (!isDuplicate) {
+        state.missingRequestStubs.push({ method, path, testName })
+    }
```

Comment:
I'm wondering if it might be worth to add the amount the duplicate has been encountered, so you also know how many times it went wrong

Reply:
Deferred for now